File: tests/machine/x/lua/python_auto.lua
local math = { sqrt = math.sqrt, pow = math.pow, sin = math.sin, log = math.log, pi = math.pi, e = math.exp(1) }
;(function(...) local parts={} for i=1,select('#', ...) do local a=select(i, ...) if a~=nil and a~='' then parts[#parts+1]=tostring(a) end end print(table.concat(parts, ' ')) end)(math.sqrt(16.0))
;(function(...) local parts={} for i=1,select('#', ...) do local a=select(i, ...) if a~=nil and a~='' then parts[#parts+1]=tostring(a) end end print(table.concat(parts, ' ')) end)(math.pi)

File: archived/tests/transpiler/x/fs/python_auto.fs
// Generated 2025-07-21 22:29 +0700

open System

module math
let pi: float = System.Math.PI
let e: float = System.Math.E
let rec sqrt x =
    System.Math.Sqrt x
let rec pow x y =
    System.Math.Pow x y
let rec sin x =
    System.Math.Sin x
let rec log x =
    System.Math.Log x

printfn "%s" (string (math.sqrt(16.0)))
printfn "%s" (string (math.pi))
